src/templates/index.development.html

```diff
@@ -5,15 +5,20 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="theme-color" content="#ffffff">
+
+
+  <title>Beenest | Book Rentals and Homes for your Business</title>
   <meta property="description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
-  <meta property="og:description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
   <meta property="og:title" content="Beenest | Book Rentals and Homes for your Business" />
-  <meta property="og:site_name" content="Beenest" />
+  <meta property="og:type" content="website" />
   <meta property="og:url" content="https://www.beenest.com" />
   <meta property="og:image" content="https://static.beenest.com/images/app/misc/host-cta-image.jpg" />
+  <meta property="og:description" content="Book rentals and homes for your next business trip. Beenest is the easiest way for business travelers to book a home or rental." />
+  <meta property="og:site_name" content="Beenest" />
```

Comment:
can the title "Beenest | Book Rentals and Homes for your Business"

Reply:
i think the title is already that

Reply:
ill chage the site_name to see if it propagates